Userland/Libraries/LibCore/MimeData.cpp

```diff
@@ -54,167 +54,129 @@ void MimeData::set_text(DeprecatedString const& text)
     set_data("text/plain", text.to_byte_buffer());
 }
 
-static AK::Array<StringView, 8> s_plaintext_suffixes = {
+// FIXME: Share this, TextEditor and HackStudio language detection somehow.
+static Vector s_plaintext_suffixes = {
```

Comment:
Probably should not be mutable? And can stay an Array (probably a `constexpr` one too)

Reply:
If we want to create every `common_extensions` inline, the variable needs to hold the data, and then it forces `s_plaintext_suffixes` to be a Vector too.
Your're right for the `const` tho.

Reply:
Vector can be constructed from a span so you can just do `.common_extensions = s_plaintext_suffixes.span()` (if it does not accept Array directly), does the same copy.